build-logic/project-plugins/src/main/kotlin/com.hedera.hashgraph.jpms-modules.gradle.kts

```diff
@@ -157,7 +157,7 @@ extraJavaModuleInfo {
         requireAllDefinedDependencies()
         requires("java.logging")
     }
-    module("com.google.guava:failureaccess", "com.google.guava.failureaccess")
+    module("com.google.guava:failureaccess", "com.google.common.util.concurrent.internal")
```

Comment:
Curious why this change was necessary.

Reply:
Google changed the package and/or module that this class is in.  Got it confirmed internally (gradle slack channel) that this is necessary.